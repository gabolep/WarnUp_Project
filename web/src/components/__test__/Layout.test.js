import react from "react";
import ReactDOM from "react-dom";
import Layout from '../Layout';
import { render, cleanup  } from '@testing-library/react'
import renderer from "react-test-renderer";


afterEach(cleanup);

it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Layout></Layout>, div)
})
it("matches snapshot", () => {
    const tree = renderer.create(<Layout/>).toJSON();
    expect(tree).toMatchSnapshot();
})