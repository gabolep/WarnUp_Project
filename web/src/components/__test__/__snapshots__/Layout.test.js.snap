// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "color": "#0A283E",
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <div
    style={
      Object {
        "width": "90%",
      }
    }
  />
</div>
`;
